doc/node.1

```diff
@@ -98,7 +98,7 @@ value to an empty string ("") disables persistent REPL history.
         type: bool  default: false
   --harmony (enable all completed harmony features)
         type: bool  default: false
-  --harmony_shipping (enable all shipped harmony fetaures)
+  --harmony_shipping (enable all shipped harmony features)
```

Comment:
This is actually auto generated from v8 flags. So, I am not sure if we should edit this. cc @yosuke-furukawa @silverwind @Fishrock123